apps/balancer-tools/src/app/poolsimulator/(components)/ShareButton.tsx

```diff
@@ -8,23 +7,22 @@ import { Tooltip } from "#/components/Tooltip";
 import { usePoolSimulator } from "#/contexts/PoolSimulatorContext";
 
 export function ShareButton() {
-  const { generateURL } = usePoolSimulator();
-  const onInitialPage = usePathname() === "/poolsimulator";
+  const { generateURL, isAnalysis } = usePoolSimulator();
   const [isNotifierOpen, setIsNotifierOpen] = useState<boolean>(false);
 
   return (
     <>
       <Tooltip
         content={
-          onInitialPage
-            ? "You can only share the analysis page"
-            : "Copy a link with the current simulation parameters"
+          isAnalysis
+            ? "Copy a link with the current simulation parameters"
+            : "You can only share the analysis page"
```

Comment:
Do people understand what the analysis page is? Not sure about our messaging around it in the app 😬